packages/prettier-plugin-java/src/printers/prettier-builder.js

```diff
@@ -0,0 +1,103 @@
+"use strict";
+const prettier = require("prettier").doc.builders;
+
+function getImageWithComments(token, toadd = "") {
+  if (
+    !token.hasOwnProperty("leadingComments") &&
+    !token.hasOwnProperty("trailingComments")
+  ) {
+    return token.image + toadd;
+  }
```

Comment:
I'm not sure of the utility of this test ?

Why do we need the toadd parameter ?

Reply:
That's some old attempts, I will remove them.